src/containers/Main.js

```diff
@@ -12,127 +12,65 @@ import Error404 from "../pages/errors/error404/Error";
-      );
-    } else {
-      return (
-        <div>
-          <HashRouter basename="/">
-            <Switch>
-              <Route
-                path="/"
-                exact
-                render={(props) => <Home {...props} theme={this.props.theme} />}
-              />
-              <Route
-                path="/home"
-                render={(props) => <Home {...props} theme={this.props.theme} />}
-              />
-              <Route
-                path="/experience"
-                exact
-                render={(props) => (
-                  <Experience {...props} theme={this.props.theme} />
-                )}
-              />
-              <Route
-                path="/education"
-                render={(props) => (
-                  <Education {...props} theme={this.props.theme} />
-                )}
-              />
-              <Route
-                path="/opensource"
-                render={(props) => (
-                  <Opensource {...props} theme={this.props.theme} />
-                )}
-              />
-              <Route
-                path="/contact"
-                render={(props) => (
-                  <Contact {...props} theme={this.props.theme} />
-                )}
-              />
-              {/* <Route
-							path="/splash"
-							render={(props) => (
-								<Splash
-									{...props}
-									theme={this.props.theme}
-								/>
-							)}
-						/> */}
-              <Route
-                path="/projects"
-                render={(props) => (
-                  <Projects {...props} theme={this.props.theme} />
-                )}
-              />
-            </Switch>
-          </HashRouter>
-        </div>
-      );
-    }
+    return (
+      <BrowserRouter basename="/">
+        <Switch>
+          <Route
+            path="/"
+            exact
+            render={(props) =>
+              settings.isSplash ? (
+                <Splash {...props} theme={this.props.theme} />
+              ) : (
+                <Home {...props} theme={this.props.theme} />
+              )
+            }
+          />
+          <Route
+            path="/home"
+            render={(props) => <Home {...props} theme={this.props.theme} />}
+          />
+          <Route
+            path="/experience"
+            exact
+            render={(props) => (
+              <Experience {...props} theme={this.props.theme} />
+            )}
+          />
+          <Route
+            path="/education"
+            render={(props) => (
+              <Education {...props} theme={this.props.theme} />
+            )}
+          />
+          <Route
+            path="/opensource"
+            render={(props) => (
+              <Opensource {...props} theme={this.props.theme} />
+            )}
+          />
+          <Route
+            path="/contact"
+            render={(props) => <Contact {...props} theme={this.props.theme} />}
+          />
+
+          {settings.isSplash && (
```

Comment:
why do we need separate route especially for splash? Isn't having splash effect on enter good enough?

Reply:
I did not separate the route. If you see the commit, there was a if else that defined two sets of the same router with the only difference being. IF splash enabled a splash route was added and other not.
Seemed redundant, so just made it singular.